cmd/mysterium_server/command_run/command.go

```diff
@@ -38,6 +38,18 @@ func (cmd *CommandRun) Run(options CommandOptions) (err error) {
 		return err
 	}
 
+	// if no identity can be selected, lets create a new one
+	if providerId == nil {
+		providerId, err = identity.CreateIdentity(options.DirectoryKeystore)
```

Comment:
What we decided on json metadata file about all local identities?
Could be like: 
```
[
    {
        "identity": "asdad12425",
        "default": false,
        "registered": true, 
    },
    {
        "identity": "asdad12425",
        "default": true,
        "registered": false, 
    }
]
```

Reply:
I thought that was just an idea. Is it confirmed now?

Reply:
1. Most important is to start having internal structure:
```
type Identity struct {
    Name string
    IsDefault boolean
    IsRegistered boolean
}
```
2. Make `dto_discovery.Identity` private OR it's not needed anymore 
3. (Optional) Metadata storage it's nice to have (can be JSON, SQL lite, etc.) we can create anotjer ticket for that